scripts/ci_build_and_test_in_container.sh

```diff
@@ -207,6 +219,16 @@ else
   fi
 fi
 
+if [[ ! -z "${SCCACHE_CREDS}" ]]; then
+  echo "sccache post-build state"
+  or_die ${SCCACHE} --show-adv-stats
+fi
```

Comment:
Maybe should we remove the `final state` lines 260-263? It's either duplicated or "false" (with code coverage disturbing the process for whatever reason)?